keylime/ca_util.py

```diff
@@ -151,6 +152,25 @@ def cmd_mkcert(workingdir: str, name: str, password: Optional[str] = None) -> No
         os.chdir(cwd)
 
 
+def cmd_import_priv(workingdir: str, priv_pem_file: str, lastserial: int) -> None:
+    cwd = os.getcwd()
+    try:
+        fs_util.ch_dir(workingdir)
+        with open(priv_pem_file, "rb") as priv_pem:
```

Comment:
If you were to read this file before changing the directory one could pass `-f privkey.pem`, otherwise one has to provide the full path.

Reply:
looking into this ...

Reply:
fixed